TerpstraSysEx.jucer

```diff
@@ -304,8 +304,8 @@
       <CONFIGURATIONS>
         <CONFIGURATION name="Debug" libraryPath="/usr/X11R6/lib/" isDebug="1" optimisation="1"
                        targetName="LumatoneSetup" defines="JUCE_MODAL_LOOPS_PERMITTED=1"/>
-        <CONFIGURATION name="Release" libraryPath="/usr/X11R6/lib/" isDebug="0" optimisation="2"
-                       targetName="LumatoneSetup" defines="JUCE_MODAL_LOOPS_PERMITTED=1"/>
+        <CONFIGURATION name="Release" libraryPath="/usr/X11R6/lib/" isDebug="0" targetName="LumatoneSetup"

```

Comment:
I removed the optimization setting in order to be consistent with the other platforms. Linux is now built with `-O3` optimization by default instead of `-O2`. The Linux binary gets slightly larger but is expected to be a little faster. If the file size is important, then activating LTO should have a stronger impact.